ui/src/router/versions/components/RouterVersionActions.js

```diff
@@ -29,9 +29,10 @@ export const RouterVersionActions = ({
         label: "Deploy",
         name: "Deploy this version",
         icon: "importAction",
-        available: config =>
-          !(isActiveConfig(config) && routerStatus !== Status.UNDEPLOYED),
-        enabled: () => routerStatus !== Status.PENDING,
+        available: () => true,
```

Comment:
This logic would mean that if all versions of a router deployment have failed, the "deploy" action is not activated for the current config. It's a minor thing but I'm wondering if the check should be `routerStatus === Status.DEPLOYED` instead of the `routerStatus !== Status.UNDEPLOYED`.

![Screenshot 2021-01-11 at 11 16 28 PM](https://user-images.githubusercontent.com/23465343/104200303-72d52980-5463-11eb-92ae-7f5cce9bbdd5.png)

Reply:
Agree

Reply:
@romanwozniak The change should be `routerStatus === Status.DEPLOYED`, not `!==`